src/pint/models/timing_model.py

```diff
@@ -2565,3 +2913,21 @@ def __str__(self):
         if self.msg is not None:
             result += "\n  " + self.msg
         return result
+
+
+class ConflictAliasError(TimingModelError):
```

Comment:
There's no need for "Error" in the name, it's an exception. And "AliasConflict" would be clearer than "ConflictAlias".